SECURITY.md

```diff
@@ -0,0 +1,18 @@
+# Security Policy
+
+## Supported Versions
+
+The versions of this project which are currently being supported with security updates are:
+
+| Version | Supported          |
+| ------- | ------------------ |
+| 1.x     | :white_check_mark: |
+
+## Reporting a Vulnerability
+
+If you find a vulnerability assosiated with this source code please contact either:
+
+- [Jarrett Alexander](mailto:alexander_jarrett@lilly.com).
+- [Nick Liffen](mailto:liffen_nicholas@lilly.com).
```

Comment:
Not going to block the fix, but these names should be updated.

Reply:
Good catch!